### ISSUE TITLE
ContentExpression is not necessarily expressed in a language

### ISSUE THREAD

**Issue by rjyounes** [semanticarts/gist#1154]
`gist:FormattedContent` is a subclass of `gist:ContentExpression` and is expressed in a `gist:MediaType`. 

`gist:ContentExpression` is expressed in a `gist:Language`.

`gist:Language` includes human and computer languages.

```
gist:FormattedContent
	a owl:Class ;
	owl:equivalentClass [
		a owl:Class ;
		owl:intersectionOf (
			gist:ContentExpression
			[
				a owl:Restriction ;
				owl:onProperty gist:isExpressedIn ;
				owl:someValuesFrom gist:MediaType ;
			]
		) ;
	] .


gist:ContentExpression
	a owl:Class ;
	rdfs:subClassOf
		[
			a owl:Restriction ;
			owl:onProperty gist:isExpressedIn ;
			owl:someValuesFrom gist:Language ;
		]
		.
```

But while some kinds of formatted content are expressed in a human language - e.g, text in HTML, PDFs, PowerPoints, plain text documents, etc. - do we consider media types like JPG and PNG to also be languages? If not, there's an error in the definitions.

**Comment by rjyounes** [comment]
Incidentally, `isExpressedIn` is defined as "The language something was expressed in," with no reference to media type.

**Comment by uscholdm** [comment]
> do we consider media types like JPG and PNG to also be languages?

I don't think so, unless it's some highly abstract notion of language.   This is what ChatGPT has to say:

You said:
**Can media types like JPG and PNG be considered as languages**

**ChatGPT said:**


No, media types like JPG and PNG are not considered languages. Instead, they are file formats or media types used for encoding digital images.
Explanation:

JPG (JPEG): Stands for Joint Photographic Experts Group, which is the organization that created the JPEG image format. It's a commonly used method of lossy compression for digital images, particularly for those images produced by digital photography. The file extension is .jpg or .jpeg.

PNG: Stands for Portable Network Graphics. It is a raster-graphics file format that supports lossless data compression. The file extension is .png.

Difference from Languages:

Languages: In computing, languages (such as programming languages like Python, markup languages like HTML, or natural languages like English) are systems of symbols and rules used to communicate information or instructions. They involve syntax, semantics, and often grammar.

File Formats: JPG and PNG, on the other hand, are not languages but are methods of encoding, storing, and transmitting image data. They define how image data should be compressed and stored in a file but do not involve syntax or grammar in the way languages do.

Summary:

    JPG and PNG: File formats/media types for images.
    Languages: Systems of symbols and rules for communication (e.g., programming languages, natural languages).

Thus, JPG and PNG are not languages; they are file formats used to store and transmit images.

**Comment by rjyounes** [comment]
That's what I thought. Therefore, the restriction on `ContentExpression` does not work for `FormattedContent`.
